odbc/src/api/get_info_tests.rs

```diff
@@ -0,0 +1,633 @@
+        expected_length = std::mem::size_of::<u16>() as i16,
+        expected_value = 0u16,
+        actual_value_modifier = modify_u16_value,
+    );
+
+    test_get_info!(
+        identifier_quote_char,
+        info_type = InfoType::IdentifierQuoteChar,
+        expected_sql_return = SqlReturn::SUCCESS,
+        buffer_length = 2,
+        expected_length = 1,
+        expected_value = "`",
+        actual_value_modifier = modify_string_value,
+    );
+
+    test_get_info!(
+        owner_term,
+        info_type = InfoType::OwnerTerm,
+        expected_sql_return = SqlReturn::SUCCESS,
+        buffer_length = 1,
+        expected_length = 0,
+        expected_value = "",
+        actual_value_modifier = modify_string_value,
+    );
+
+    test_get_info!(
+        catalog_name_separator,
+        info_type = InfoType::CatalogNameSeparator,
+        expected_sql_return = SqlReturn::SUCCESS,
+        buffer_length = 2,
+        expected_length = 1,
+        expected_value = ".",
+        actual_value_modifier = modify_string_value,
+    );
+
+    test_get_info!(
+        catalog_term,
+        info_type = InfoType::CatalogTerm,
+        expected_sql_return = SqlReturn::SUCCESS,
+        buffer_length = 9,
+        expected_length = 8,
+        expected_value = "database",
+        actual_value_modifier = modify_string_value,
+    );
+
+    test_get_info!(
+        convert_functions,
+        info_type = InfoType::ConvertFunctions,
+        expected_sql_return = SqlReturn::SUCCESS,
+        expected_length = std::mem::size_of::<u32>() as i16,
+        expected_value = 2u32,
+        actual_value_modifier = modify_u32_value,
+    );
+
+    test_get_info!(
+        numeric_functions,
+        info_type = InfoType::NumericFunctions,
+        expected_sql_return = SqlReturn::SUCCESS,
+        expected_length = std::mem::size_of::<u32>() as i16,
+        expected_value = 7663201u32,
+        actual_value_modifier = modify_u32_value,
+    );
+
+    test_get_info!(
+        string_functions,
+        info_type = InfoType::StringFunctions,
+        expected_sql_return = SqlReturn::SUCCESS,
+        expected_length = std::mem::size_of::<u32>() as i16,
+        expected_value = 16254993u32,
+        actual_value_modifier = modify_u32_value,
+    );
+
+    test_get_info!(
+        system_functions,
+        info_type = InfoType::SystemFunctions,
+        expected_sql_return = SqlReturn::SUCCESS,
+        expected_length = std::mem::size_of::<u32>() as i16,
+        expected_value = 0u32,
+        actual_value_modifier = modify_u32_value,
+    );
+
+    test_get_info!(
+        timedate_functions,
+        info_type = InfoType::TimedateFunctions,
+        expected_sql_return = SqlReturn::SUCCESS,
+        expected_length = std::mem::size_of::<u32>() as i16,
+        expected_value = 1572864u32,
+        actual_value_modifier = modify_u32_value,
+    );
+
+    test_get_info!(
+        convert_big_int,
+        info_type = InfoType::ConvertBigInt,
```

Comment:
Since this test and the following until `InfoType::ConvertGuid` have the same expected values, I think it would help with readability if we define another macro for these `SQL_U32_ZERO` expected values.  Something like:
```
test_get_info_expect_u32_zero! (convert_big_int, info_type = InfoType::ConvertBigInt);
test_get_info_expect_u32_zero! (convert_decimal, info_type = InfoType::ConvertDecimal);
test_get_info_expect_u32_zero! (convert_double, info_type = info_type = InfoType::ConvertDouble);
...
```

Reply:
good idea -- done